src/formatDate.js

```diff
@@ -50,7 +50,53 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const currentSeparator = fromFormat.slice(-1);
+  const parsedDate = date.split(currentSeparator);
+
+  const newSeparator = toFormat.slice(-1);
+  const toFormatOrder = toFormat.slice(0, 3);
+  const formattedDate = [];
+
+  const YEAR_BREAKPOINT = 30;
+  const THIS_CENTURY = 20;
+  const LAST_CENTURY = 19;
+  const SHORT_DATE = 2;
+  const LONG_DATE = 4;
```

Comment:
Why are you declaring these variables in upper case?

Reply:
I  just left inside only variables that cannot be taken out. I'm not 100% sure where is the best place to declare these 4, but I hope, next reviewer will describe the best options